### ISSUE TITLE
what's the right install location for vim ?

### ISSUE THREAD

**Issue by lookforit** [angelozerr/tern-node-express#17]
I use `tern_for_vim` and `youcompleteme` for code auto-complete.
I installed `tern-node-express` in `~/.vim/bundle/tern_for_vim/node_modules`,but object members(app.set) is not listed in the menu .

Did I install it correctly?

**Comment by angelozerr** [comment]
If you have a tern folder `~/.vim/bundle/tern_for_vim/node_modules/tern-node-express/node_modles/tern`, try to remove it at hand.

**Comment by lookforit** [comment]
yes,it seems working partly.The member functions of `app` can be listed out after `dot` typing.
But the `res` 's member functions still don't come out.

**Comment by Viterzgir** [comment]
I'm also have problem with tern-node-express. I can't figure out why autocomlete is not working in my instalation. Anyone know how I can see log or some debug information?

**Comment by thejmazz** [comment]
I was able to get it working with `YouCompleteMe` (and no `tern_for_vim`) - I had to delete `tern` from `tern-node-express`'s node_modules as per above:

``` bash
rm -rf ~/.vim/bundle/YouCompleteMe/third_party/ycmd/third_party/tern_runtime/node_modules/tern-node-express/node_modules/tern
```

Also this is my `~/.tern-config`:

``` json
{
  "plugins": {
    "node": {},
    "node-express": {}
  }
}
```

See my [Dockerfile](Dockerfile) for the complete build.

**Comment by angelozerr** [comment]
I would like to try to update package.json with peerDependencies to avoid this problem.

**Comment by sirusdas** [comment]
Hello, 
.tern-config code is 
```
{
  "plugins": {
    "node": {},
    "node-express": {},
    "es_modules": {},
    "angular": {}
  },
  "libs": [
      "ecma5",
      "ecma6",
      "browser",
      "underscore",
      "jquery"
    ],
    "ecmaVersion": 6
}

```
I have node project installed in other directory where i have created .tern-project file and added the same code above. Also installed tern and tern-node-express in both 
~/.vim/bundle/tern_for_vim/node_modules/tern-node-express/
and
in my project direcory node_modules has both tern and tern-node-express

I still have no success in using express. Can anyone help?

**Comment by Luffy4321** [comment]
I use tern_for_vim and youcompleteme for code auto-complete. express cannot work correctly。
my pc is MacBookPro14,3 macOS Mojave。
How to  install it correctly?Thanks in advance！

**Comment by jabby** [comment]
Hello @Luffy4321 
I don't know how to install tern_for_vim
There is an alternative with language server. One of my friend use this config. https://github.com/charlyx/vim-config/blob/master/init.vim
Maybe it will help you for express completion.
